dockerfiles/ws-health-exporter/README.md

```diff
@@ -0,0 +1 @@
+A health checker for Substrate-based nodes. See source https://gitlab.parity.io/parity/infrastructure/ansible/collections/chain_operations/-/raw/master/roles/ws_health_exporter/files/exporter.py
```

Comment:
Better to post link to the source of the issue https://github.com/paritytech/substrate/issues/1017, since this is public repo.
ws-health-exporter is a temporary solution until proper reediness probe will be build inside substrate.